opencga-app/src/main/java/org/opencb/opencga/app/cli/main/executors/JobsCommandExecutor.java

```diff
@@ -282,14 +321,14 @@ private RestResponse<Job> retry() throws Exception {
             PrintUtils.println(getObjectAsJSON(jobRetryParams));
             return res;
         } else if (commandOptions.jsonFile != null) {
-            jobRetryParams = JacksonUtils.getDefaultObjectMapper()
-                    .readValue(new java.io.File(commandOptions.jsonFile), JobRetryParams.class);
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This should be a "readValue" instead of a "writeValue"
We are losing this changes: https://github.com/opencb/opencga/pull/1916